scripts/data_utils.py

```diff
@@ -36,11 +38,226 @@
 SENTENCE_ENDINGS = [".", "!", "?"]
 WORDS_BREAKS = list(reversed([",", ";", ":", " ", "(", ")", "[", "]", "{", "}", "\t", "\n"]))
 
+HTML_TABLE_TAGS = {"table_open": "<table>", "table_close": "</table>", "row_open":"<tr>"}
+
 PDF_HEADERS = {
     "title": "h1",
     "sectionHeading": "h2"
 }
 
+class TokenEstimator(object):

```

Comment:
modify exisitng token estimator instead of using new one https://github.com/microsoft/sample-app-aoai-chatGPT/blob/bb16ff28f897c93b4f7df1850e48687412c185f9/scripts/data_utils.py#L271C30-L271C30